curiouslearning/FeedTheMonsterJS
# feat: adjust background elements and fix tab issues and updated the stone drop coordinates (FM-295)

# Changes
- Adjust background elements using new set of images and fix tablet issues found and adjust stone drop coordinate

# How to test
- Open the start screen of FTM.
- Tap or click any part of the start screen to proceed to the next screen
- Select any level to play
- make sure the background elements are displaying properly both tab and mobile

Ref: [FM-295](https://curiouslearning.atlassian.net/browse/FM-295)


[FM-295]: https://curiouslearning.atlassian.net/browse/FM-295?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
